docs/tutorials.rst

```diff
@@ -0,0 +1,24 @@
+Tutorials
+=========
+
+Open edX customization
+----------------------
+
+.. toctree::
+   :maxdepth: 2
+
+   tutorials/theming
+   tutorials/edx-platform-settings
+   tutorials/edge
+
+System administration
+---------------------
+
+.. toctree::
+   :maxdepth: 2
+
+   tutorials/portainer
+   tutorials/proxy
+   tutorials/datamigration
+   tutorials/multiplatforms
+   tutorials/oldreleases
```

Comment:
Should "Running Tutor with Podman" also be in this section?

Reply:
Yes, you're right.

Reply:
I attempted to actually run Tutor with Podman and I was sorely disappointed. The only reliable source of docs that I found concerning the integration with docker-compose is this blog post: https://www.redhat.com/sysadmin/podman-docker-compose
There are no other official docs :sweat: 

1. The instructions given in the blog post don't work out of the box. Launching the podman service failed altogether on Ubuntu 20.04 and 20.10. It worked on CentOS 8, but some parameters need to changed, such as the docker socket path.
2. After I got the podman service working, I managed to get an Open edX platform running with tutor, but with the root user. Then, containers complained that they could not write data to the bind-mounted volumes. I attempted to run as a non-root user, and discovered that the podman socket is only readable by root. This should explain why all commands from that blog post are prefixed by `sudo`.

Long story short, I was hoping to update the tutorial. Instead, I'll just move it for the sake of better organisation. For the life of me, I do not understand why some people would want to run Podman instead of Docker. Bad documentation is an immediate turn-off for me.